website/docs/guides/1.10.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.10.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1100-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.10.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.10.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.10.0 has a number of new and exciting features and changes.
+
+**New Resources, Data Sources, and Features:**
+- You can now manage [Atlas Organizations](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Organizations/operation/createOrganization) including Create with [`mongodbatlas_organization`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization)
```

Comment:
We may want to note that this is for customers with cross organization billing enabled with a link.

Reply:
added